src/forms/RadioButtonGroup.jsx

```diff
@@ -37,12 +37,12 @@ export default class RadioButtonGroup extends PureComponent {
 	}
 
 	render() {
-		const { direction, switchDirection, children, name, className } = this.props;
+		const { direction, switchDirection, children, name, className, wrap } = this.props;
 
 		const switchDirectionAttr = switchDirection ? { switchDirection } : '';
 
 		return (
-			<Flex direction={direction} {...switchDirectionAttr} className={className}>
+			<Flex direction={direction} {...switchDirectionAttr} className={className} wrap={wrap}>
```

Comment:
odd there was no test that was updated :(

Reply:
Fixed!